lib/widgets/compose_box.dart

```diff
@@ -181,6 +251,8 @@ class ComposeContentController extends ComposeController<ContentValidationError>
       if (textNormalized.length > kMaxMessageLengthCodePoints)
         ContentValidationError.tooLong,
 
+      if (_quoteAndReplies.isNotEmpty) ContentValidationError.quoteAndReplyInProgress,
```

Comment:
nit:
```suggestion
      if (_quoteAndReplies.isNotEmpty)
        ContentValidationError.quoteAndReplyInProgress,
```

for consistency with the others around it